.env.example

```diff
@@ -26,4 +26,5 @@ NORTHSTAR_URI=http://northstar.dev:8000
 NORTHSTAR_API_KEY=abc4324
 
 PHOENIX_URI=https://dev.dosomething.org:8888
+PHOENIX_API_URI=https://staging.beta.dosomething.org/api
```

Comment:
hmm, i think staging just so it works out of the box. if someone needs to go gladiator / phoenix  dev then they can prob change the URL